src/student/student.service.ts

```diff
@@ -1,4 +1,34 @@
 import { Injectable } from '@nestjs/common';
+import { Student } from './student.entity';
 
 @Injectable()
-export class StudentService {}
+export class StudentService {
+  async getStudents(status: number): Promise<Student[]> {
+    return await Student.find({
+      where: {
```

Comment:
W tej metodzie też zastosował bym select (id, firstName, lastName + ewentualnie pole do wyciągnięcia avatara dla widoku https://xd.adobe.com/view/864faeb9-d762-4277-a5d1-5b7565dcf543-d31c/screen/faa8be4a-a848-43db-8a13-c7d3e2c1e604/?utm_campaign=WYNIKI+TWOJEGO+ZALICZENIA+MEGAK+-+GRATULACJE&utm_medium=email&utm_source=newsletter&utm_term=2023-04-18 )
Z tego co patrzę na szybko to powinniśmy to robiić na podstawie githubUsername
https://docs.google.com/document/d/1j3iltSfaJXB8lVi5dwApL9UU0ze7A8kz9DBDChIVwfw/edit#heading=h.upi0uw6apyak